openage/convert/value_object/read/media/smx.pyx

```diff
@@ -319,6 +734,21 @@ cdef class SMXLayer:
     # pixel matrix representing the final image
     cdef vector[vector[pixel]] pcolor
 
+    # memory pointer
+    cdef const uint8_t[::1] data_raw
+
+    # current command
+    cdef int cmd_offset
+
+    # current color
+    cdef int color_offset
+
+    # current chunk position
+    cdef int chunk_pos
+
+    # rows
+    cdef size_t row_count
```

Comment:
Is there a reason why we would want to promote these variables to members instead of keeping them local? All of these members are never accessed again after the layer has been read. Keeping `data_raw` around is especially resource-intensive because we would store the whole original graphics file in memory well after it was used.